### PR TITLE
WIP: Ana/create addresses for different networks

### LINKED ISSUE #3316: Create addresses for different networks
It should be possible to create an address for different networks apart from Cosmos.

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #3316 

**Description:**
Work in progress, just added the network prefix to the `getNewWalletFromSeed` function. Need to check transactions and so on.

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/vuex/modules/keystore.js [hunk 1]
```diff
@@ -16,6 +16,11 @@ export default () => {
     }
   }
 
+  const networkPrefix = localStorage
```
Comment (faboweb) [inline src/vuex/modules/keystore.js]: too much magic. this should be a part of the network object in the API. i.e. `bech32prefix`

Comment (Bitcoinera) [reply src/vuex/modules/keystore.js]: Oh, I love this idea! Thank you! :+1:

**Comment by faboweb** [comment]
check #3337 

sorry wanted to do it

**Comment by Bitcoinera** [comment]
> check #3337
> 
> sorry wanted to do it

:crying_cat_face:

**Pull request closed without merging**